### PR TITLE
[DC-2617] Create cleaning rule to update observation_concept_id and value_as_concept_id

### PR DESCRIPTION
**Pull request by hiro-mishima**

### CODE REVIEW

File: data_steward/cdr_cleaner/cleaning_rules/convert_pre_post_coordinated_concepts.py [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+        Queries are generated for creating a sandbox table, deleting records from observation,
+        and inserting new records to observation.
+
+        The sandbox query sandboxes rows from observation that meet the following criteria:
+            1. value_source_concept_id is a non-standard concept, and
+            2. value_source_value has a "Maps to value" relationship.
+
+        The delete query deletes rows from observation that are sandboxed.
+
+        The insert query isnerts rows with newly assigned observation_ids and updated
```
Comment (brendagutman) [inline data_steward/cdr_cleaner/cleaning_rules/convert_pre_post_coordinated_concepts.py:164]: These are great descriptions! Thanks for making these observations distinguishable from the backfill_skip observations as well as making it all clear in the note. 

There is a tiny typo here.

**Review by brendagutman — APPROVED** [review]
It all looks great. Thanks for putting in great work Hiro.

File: data_steward/cdr_cleaner/cleaning_rules/convert_pre_post_coordinated_concepts.py [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+            'post-coordinated mapping.')
+
+        super().__init__(issue_numbers=JIRA_ISSUE_NUMBERS,
+                         description=desc,
+                         affected_datasets=[cdr_consts.RDR],
+                         affected_tables=[OBSERVATION],
+                         project_id=project_id,
+                         dataset_id=dataset_id,
+                         sandbox_dataset_id=sandbox_dataset_id,
+                         depends_on=[SetConceptIdsForSurveyQuestionsAnswers],
```
Comment (lrwb-aou) [inline data_steward/cdr_cleaner/cleaning_rules/convert_pre_post_coordinated_concepts.py:150]: I would suggest expanding the depends on to include the `UpdateFamilyHistoryCodes` rule.

File: data_steward/cdr_cleaner/cleaning_rules/convert_pre_post_coordinated_concepts.py (hunk 1 above)
Comment (hiro-mishima) [reply data_steward/cdr_cleaner/cleaning_rules/convert_pre_post_coordinated_concepts.py]: Thanks! I fixed the typo.

File: data_steward/cdr_cleaner/cleaning_rules/convert_pre_post_coordinated_concepts.py (hunk 2 above)
Comment (hiro-mishima) [reply data_steward/cdr_cleaner/cleaning_rules/convert_pre_post_coordinated_concepts.py]: Added `UpdateFamilyHistoryCodes` as well :)

**Pull request merged**